### PR TITLE
K8S-381 Connect all LXC services (UI/ETS/etc) to RPC-O MaaS

### PR DESCRIPTION
**Pull request by xgerman**
Adds process checks and (local) api checks.

Note: Since most of the services except ui should not be widely exposed
I am not adding checks for reaching them from the Interweb.

### CODE REVIEW

**Comment by xgerman** [comment]
needs testing

**Comment by npawelek** [comment]
@xgerman All of the new templates need to be updated to expose metadata per PR #499.

**Comment by npawelek** [comment]
@xgerman Sorry to be nit-picky, but can you move these lines to right above `alarms`? That way it is consistent with *all* templates.

**Comment by tonytan4ever** [comment]
Needs a rebase.
Also the code 

> {% from "templates/common/macros.jinja" import get_metadata with context %}

Needs to be put in the first line of all the ticket.

File: releasenotes/notes/K8S-381_monitor_mk8s_containers-1a478428700d9039.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Managed Kubernetes instals several laxc-conatiners for various
```
Comment (CrashenX) [inline releasenotes/notes/K8S-381_monitor_mk8s_containers-1a478428700d9039.yaml]: I didn't actually review this, I was just scrolling through quickly and my eyes caught this typo: s/instals/installs

Comment (CrashenX) [reply releasenotes/notes/K8S-381_monitor_mk8s_containers-1a478428700d9039.yaml]: Also, is laxc a thing, or is it lxc?

**Comment by xgerman** [comment]
recheck CIT/xenial_pike_deploy

**Comment by tonytan4ever** [comment]
@xgerman: Overall looks good.  Just need rebase and gates. Sorry, rebase one more time is needed.

**Pull request merged**